### PR TITLE
Bump hip clang version

### PR DESCRIPTION
**Pull request by daniellowell**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -111,15 +111,15 @@ def buildHipClangJob(compiler, flags, image, prefixpath="/opt/rocm", cmd = ""){
             withDockerContainer(image: image, args: dockerOpts) {
                 timeout(time: 5, unit: 'MINUTES')
                 {
-                    sh 'PATH="/opt/rocm/opencl/bin/x86_64/:$PATH" clinfo'
+                    sh 'PATH="/opt/rocm/opencl/bin/:$PATH" clinfo'
```
Comment (pfultz2) [inline Jenkinsfile:114]: I am not sure if this will work in both rocm containers. Maybe we should just use the `clinfo` provided by ubuntu.

Comment (daniellowell) [reply Jenkinsfile:114]: We can just add both paths. I’ve had issues with the base clinfo, but that might just be me.

**Comment by daniellowell** [comment]
On hold until we have a hip-clang build that correctly point clang-ocl to `/opt/rocm/lib`

File: hip-clang.docker [hunk 2]
```diff
@@ -6,8 +6,7 @@ ARG PREFIX=/usr/local
 RUN dpkg --add-architecture i386
 
 # Add rocm repository
-RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://compute-artifactory.amd.com/artifactory/list/rocm-osdb-deb/ compute-rocm-dkms-no-npi-hipclang 1927 > /etc/apt/sources.list.d/rocm.list'
-
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://compute-artifactory.amd.com/artifactory/list/rocm-osdb-deb/ compute-rocm-dkms-no-npi-hipclang 2272 > /etc/apt/sources.list.d/rocm.list'
```
Comment (atamazov) [inline hip-clang.docker:9]: +1 as currently I see some strange failures with `comgr-attach` branch and suspect that comgr is not fully functional in 1927.

**Pull request merged**